src/MetaGraphs.jl

```diff
@@ -368,6 +372,10 @@ function default_index_value(v::Integer, prop::Symbol, index_values::Set{Any}; e
     return val
 end
 
+"Checks whether `prop` with `val` is available to be used as indexing for node `v`"
+function index_available(g::AbstractMetaGraph, v::Integer, prop::Symbol, val::Any)
+    !in(prop, g.indices) || !haskey(g.metaindex[prop], val) || (haskey(g.vprops, v) && haskey(g.vprops[v], prop) && g.vprops[v][prop] == val)
```

Comment:
Maybe clarify the order of binary boolean operators? This is not very readable for an outsider

Reply:
@gdalle hi, could you please elaborate more what you would need ?

Reply:
thanks for the clarification!